### PR TITLE
Fix up to publish as W3C Note. 

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,130 +6,115 @@
...
     <script src='http://www.w3.org/Tools/respec/respec-w3c-common' async=""
     class='remove'>
 </script>
     <script class='remove'>
 /*Respec config*/
-    var respecConfig = {
-	additionalCopyrightHolders: "Copyright © the Contributors to this Specification, published by the <a href='http://www.w3.org/community/respimg/'>Responsive Images Community Group</a> under the <a href= 'https://www.w3.org/community/about/agreements/cla/'>W3C Community Contributor License Agreement (CLA)</a>. A human-readable <a href= 'http://www.w3.org/community/about/agreements/cla-deed/'>summary</a> is available.",
+    var respecConfig = { 
+    noLegacyStyle: true,
+    //additionalCopyrightHolders: "Copyright © the Contributors to this Specification, published by the <a href='http://www.w3.org/community/respimg/'>Responsive Images Community Group</a> under the <a href= 'https://www.w3.org/community/about/agreements/cla/'>W3C Community Contributor License Agreement (CLA)</a>. A human-readable <a href= 'http://www.w3.org/community/about/agreements/cla-deed/'>summary</a> is available.",
```
Comment (nwtn) [inline index.html:22]: why remove the copyright statement?

File: index.html [hunk 2]
```diff
@@ -454,856 +432,153 @@
           attributes</a>
         </dd>
         <dd>
-          <code title="attr-picture-alt"><a href=
-          "#attr-picture-alt">alt</a></code>
```
Comment (nwtn) [inline index.html:458]: No `alt`?

Comment (marcoscaceres) [inline index.html:458]: Nope, no `alt`. Use actual HTMLs. Alt is a poor substitute for HTML elements.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html:22]: It's under the w3c copyright now.

File: index.html [hunk 3]
```diff
@@ -161,9 +148,12 @@
...
       <p class="note">
-        This specification is under active development and changing frequently.
-        Please see the list of <a href="#dfn-open-issues" class=
-        "internalDFN">open issues</a>.
+        Work on this specification has stopped as it was superceded by the
+        <a href=
+        "http://tabatkins.github.io/specs/respimg/Overview.html">RespImg Syntax
+        proposal</a>. The mailing list gives the justification as to why we
+        stopped persuing the standardization of the <code>picture</code>
+        element.
```
Comment (marcoscaceres) [inline index.html]: @Wilto still needs to provide us a link here. Will link to http://lists.w3.org/Archives/Public/public-respimg/2013Oct/0020.html for now.

**Comment by nwtn** [comment]
I'm satisfied. LGTM.

**Pull request merged**